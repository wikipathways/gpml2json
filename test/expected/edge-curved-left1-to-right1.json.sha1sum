dfdb32ac4c8306275c719c46bd208426181693c3 ?-
