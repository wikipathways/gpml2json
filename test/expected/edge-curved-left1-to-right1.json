{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["c9937","fe768","d5dea"],"drawAs":"rect","gpmlElementName":"Pathway","height":818.1126371280204,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":1047.7865523249097,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/4abdbd60bb83021a3d74aae01e065aefc79c93ba9360d8fd4eb8eab2bbf88b84"},"entityMap":{"fe768":{"id":"fe768","textContent":"0018","wpType":"GeneProduct","x":19.553949158140107,"y":60.55728028355435,"fontSize":10,"height":31.57173175615435,"verticalAlign":"middle","width":87.07031192659016,"zIndex":32963,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"d5dea":{"id":"d5dea","textContent":"GeneProduct","wpType":"GeneProduct","x":276.97689917295247,"y":133.41602310076962,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32964,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"c9937":{"id":"c9937","drawAs":"CurvedLine","borderWidth":1,"zIndex":32962,"points":[{"isAttachedTo":"d5dea","attachmentDisplay":{"position":[0,0.5],"offset":[0,0]},"x":276.97689917295247,"y":143.41602310076962,"orientation":[-1,0]},{"y":109.87958463120057,"x":256.97689917295247},{"isAttachedTo":"fe768","attachmentDisplay":{"position":[1,0.5],"offset":[0,0]},"x":106.62426108473028,"y":76.34314616163152,"orientation":[-1,0]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["d5dea","fe768"]}}}
